src/screens/Tasks/components/SearchBar.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react-hooks/exhaustive-deps */
+import React, { useState, useEffect } from "react";
+import { Form } from "react-bootstrap";
+import useDebounce from "~/hook/useDebounce";
+
+interface SearchBarProps {
+	onSearch: (query: string) => void;
+	onReset: () => void;
+	placeholder?: string;
+	debounceTime?: number;
+}
+
+const SearchBar: React.FC<SearchBarProps> = ({
+	onSearch,
+	onReset,
+	placeholder,
+	debounceTime = 500,
+}) => {
+	const [searchTerm, setSearchTerm] = useState("");
```

Comment:
Sửa các thằng liên quan
```suggestion
	const [searchValue, setSearchValue] = useState("");
```